app/controllers/serviceLearning/routes.py

```diff
@@ -185,3 +189,26 @@ def renewCourse(courseID, termID):
         flash("Renewal Unsuccessful", 'warning')
 
     return "", 500
+
+@serviceLearning_bp.route('/serviceLearning/sendRecommendation/<termID>', methods = ['GET'])
+def sendRecommendation(termID):
+    """
+    This function allows the download of csv file
+    """
+    try:
+        designator = "sendRecommendation"
+        csvInfo = approvedCourses(termID)
+        fileFormat = {"headers":["Course Name", "Course Number", "Faculty", "Term"]}
+        filePath = safe_join(os.getcwd(), app.config['files']['base_path'])
+        # print(filePath)
+        newFile = fileMaker(designator, csvInfo, "CSV", fileFormat)
+        print(newFile.relativePath)
+        # cwd = os.getcwd()
+        # print(os.getcwd())
+        # print(app.config['files']['base_path'])
+        # return send_from_directory(filePath, 'ApprovedCourses.csv')
```

Comment:
Removed the comments. Added the column in the downloaded form but left the status of previously approved courses blank for now because the merge in development for tracking previously approved courses is not done yet.

Reply:
I added the text of the body and subject when sending recommendation